### PR TITLE
:zap: Added GridMask Augmentation

### PR DESCRIPTION
**Pull request by kartik4949**
Hi, @mingxingtan 

I Have Added GridMask Augmentation and tested it on some sample data
GridMask Augmentation is one of SOTA augmentation for Computer Vision
Refer - https://arxiv.org/abs/2001.04086
For Results Refer - https://paperswithcode.com/paper/gridmask-data-augmentation/review/
They got 1.8+ increase in MAP for FastRcnn

The Code I Develop is in Core tensorflow with tf.function wrapper so it will run in static graph mode with performance boost.
Please test the result with this PR with finetuned hyperparametres.
Thanks

### CODE REVIEW

File: efficientdet/hparams_config.py [hunk 1]
```diff
@@ -182,6 +182,7 @@ def default_detection_configs():
   h.jitter_max = 2.0
   h.autoaugment_policy = None
   h.use_augmix = False
+  h.grid_mask = False
```
Comment (kartik4949) [inline efficientdet/hparams_config.py:185]: @mingxingtan Do i need to add configuration for gridmask like augmix?

Comment (mingxingtan) [reply efficientdet/hparams_config.py:185]: Thank you, @kartik4949 
Having this flag is good enough.  Do you find this augmentation helpful?

**Review by mingxingtan — APPROVED** [review]

File: efficientdet/hparams_config.py (hunk 1 above)
Comment (kartik4949) [reply efficientdet/hparams_config.py:185]: @mingxingtan you can refer to the above links 
And please test this pr with your data or sample data ,Im short on resources so cant train but i have checked the augmentation for 2-3 images!

Comment (mingxingtan) [reply efficientdet/hparams_config.py:185]: ok, I will merge this PR and give it a try on COCO later.

Comment (kartik4949) [reply efficientdet/hparams_config.py:185]: @mingxingtan yes please do try it with different hyperparameters :)

**Pull request merged**